vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -29,6 +29,11 @@
   owl:imports <http://qudt.org/2.1/vocab/quantitykind> ;
   owl:imports <http://qudt.org/2.1/vocab/sou> ;
   owl:versionIRI <http://qudt.org/2.1/vocab/unit> ;
+  dcterms:rights """
```

Comment:
This would be better placed in the metadata instance: vaem:GMD_QUDT-UNITS-ALL later in this file.

Also, could you do the same in the QUDT Schema (http://qudt.org/2.1/schema/qudt), again in the metadata instance?

Thanks.

Reply:
Done.